sycl/include/CL/sycl/half_type.hpp

```diff
@@ -127,9 +127,9 @@ template <> struct hash<half> {
 
 #define SYCL_HLF_MAX_EXP 16
 
-#define SYCL_HLF_MIN_10_EXP -4
+#define SYCL_HLF_MIN_10_EXP (-4)
 
-#define SYCL_HLF_MIN_EXP -13
+#define SYCL_HLF_MIN_EXP (-13)
```

Comment:
This 2 cases are not macro parameters and I do not see the motivation for this.

Reply:
Sorry, I got this patch from clang-tidy and missed that one rule did unrelated fixes. I assume it puts parenthesis to avoid confusion of unary minus with binary minus or maybe problems with expression evaluation order.
Do you want me to revert this change?